pagerduty/resource_pagerduty_incident_workflow.go

```diff
@@ -0,0 +1,418 @@
+package pagerduty
+
+import (
+	"context"
+	"log"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/heimweh/go-pagerduty/pagerduty"
+)
+
+func resourcePagerDutyIncidentWorkflow() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   resourcePagerDutyIncidentWorkflowRead,
+		UpdateContext: resourcePagerDutyIncidentWorkflowUpdate,
+		DeleteContext: resourcePagerDutyIncidentWorkflowDelete,
+		CreateContext: resourcePagerDutyIncidentWorkflowCreate,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		CustomizeDiff: customizeIncidentWorkflowDiff(),
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Please consider including the default description...
`Default:  "Managed by Terraform",`

Reply:
Good idea. Done.